asati/cli.py

```diff
@@ -20,7 +24,67 @@ def main(*, verbose: bool) -> None:
     logger.info(configure_sentry())
     logger.info("Running process")
 
-    # Do things here!
+    ssm_client = SSMClient()
```

Comment:
Commenting here as high up in the `main()` CLI function as possible.

I think I'd find some comments in here helpful.  I think I'm okay without breaking this out into helper functions, but it's a bit dense without some navigational aides.

Example:

```python
# get last accession id from SSM parameter
ssm_client = SSMClient()
last_accession_parameter = os.environ["LAST_ACCESSION_PARAMETER"]
last_accession_uri_id = ssm_client.get_parameter(last_accession_parameter)
logger.info(
    f"SSM parameter '{last_accession_parameter}' retrieved: '{last_accession_uri_id}'"
)

# retrieve list of all accessions from ArchivesSpace
asnake_client = ASnakeClient(
    baseurl=os.environ["ARCHIVESSPACE_URL"],
    username=os.environ["ARCHIVESSPACE_USER"],
    password=os.environ["ARCHIVESSPACE_PASSWORD"],
)
accession_uri_ids = asnake_client.get(
    "/repositories/2/accessions?all_ids=true"
).json()

# exit early if no new accessions to process
if max(accession_uri_ids) <= int(last_accession_uri_id):
    logger.info("No new accessions to add to Airtable.")
    return

# and so forth...
```

Reply:
Good call!